src/cdomain/value/cdomains/lval.ml

```diff
@@ -1,3 +1,672 @@
 (** Domains for {!GoblintCil.lval}. *)
```

Comment:
Where is all this commented out code coming from? Is this an artifact of a merge?